src/test/java/core/basesyntax/dao/StorageDaoTest.java

```diff
@@ -0,0 +1,86 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.storage.Storage;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class StorageDaoTest {
+    private static Fruit banana;
+    private static StorageDao storageDao;
+
+    @BeforeClass
+    public static void beforeClass() {
+        banana = new Fruit("Banana");
+        storageDao = new StorageDaoImpl();
+    }
+
+    @Test
+    public void add_ok() {
+        storageDao.add(banana, 10);
+        int expected = 10;
+        int actual = Storage.fruitStorage.get(banana);
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void addMore_ok() {
+        Storage.fruitStorage.put(banana, 10);
+        storageDao.add(banana, 20);
+        int expected = 30;
+        int actual = Storage.fruitStorage.get(banana);
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void add_sizeAfter_ok() {
+        storageDao.add(banana, 10);
+        int size = Storage.fruitStorage.size();
+        Assert.assertEquals(1, size);
+    }
+
+    @Test
+    public void add_sizeAfterMore_ok() {
+        Storage.fruitStorage.put(banana, 15);
+        storageDao.add(banana, 10);
+        int size = Storage.fruitStorage.size();
+        Assert.assertEquals(1, size);
+    }
+
+    @Test
+    public void remove_ok() {
+        Storage.fruitStorage.put(banana, 10);
+        storageDao.remove(banana, 5);
+        int expected = 5;
+        int actual = Storage.fruitStorage.get(banana);
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void get_ok() {
+        Storage.fruitStorage.put(banana, 10);
+        int expected = 10;
+        int actual = storageDao.get(banana);
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void getAll() {
```

Comment:
You have to follow test mehods naming convention. Use this pattern:
`methodWeTesting_inputState_ExpextedBehavior() `
Be attentive to capital letters. We use here:
`lowerCamelCase_lowerCamelCase_UpperCamelCase()`
*Use this pattern to every test method name